tests/test_routes.py

```diff
@@ -0,0 +1,42 @@
+def test_get_all_planets_with_no_records(client):
```

Comment:
I would also include a get_all _planets test with several planets in the DB as well.